src/ProjectTemplates/test/SpaTemplateTest/SpaTemplateTestBase.cs

```diff
@@ -240,6 +240,20 @@ private void TestBasicNavigation(bool visitFetchData, bool usesAuth, IWebDriver
                     browser.FindElement(By.Name("Input.Password")).SendKeys(password);
                     browser.FindElement(By.Name("Input.ConfirmPassword")).SendKeys(password);
                     browser.FindElement(By.Id("registerSubmit")).Click();
+
+                    // We will be redirected to the RegisterConfirmation
+                    browser.Contains("/Identity/Account/RegisterConfirmation", () => browser.Url);
+                    browser.FindElement(By.PartialLinkText("Click here to confirm your account")).Click();
+
+                    // We will be redirected to the ConfirmEmail
+                    browser.Contains("/Identity/Account/ConfirmEmail", () => browser.Url);
+
+                    // Now we can login
+                    browser.FindElement(By.PartialLinkText("Login")).Click();
+                    browser.Exists(By.Name("Input.Email"));
+                    browser.FindElement(By.Name("Input.Email")).SendKeys(userName);
+                    browser.FindElement(By.Name("Input.Password")).SendKeys(password);
+                    browser.FindElement(By.Id("login-submit")).Click();
```

Comment:
@javiercn I must be doing something slightly wrong in updating the test, now that confirmation is required I had to add a few more clicks before the account should be logged in, but it doesn't look like the fetch data works still, anything obviously wrong with this test update?

Reply:
Can it be that the return url is getting lost in the flow and is not redirecting back to fetch-data?

I would recommend you try the flow manually with the latest preview9

Reply:
Was just missing a click on the fetch data at the end I guess, thanks